certs/certs.c

```diff
@@ -28,7 +31,10 @@ const char certificates[] =
 "ksLi4xaNmjICq44Y3ekQEe5+NauQrz4wlHrQMz2nZQ/1/I6eYs9HRCwBXbsdtTLS\r\n"
 "R9I4LtD+gdwyah617jzV/OeBHRnDJELqYzmp\r\n"
 "-----END CERTIFICATE-----\r\n"
-/*DigiCert Global Root CA*/
+#endif /* BALTIMORE_CERT */
+
+#if !(defined(USE_BALTIMORE_CERT_ONLY) || defined(USE_GERMANY_CERT_ONLY))
+/*DigiCert Global Root CA --China--*/
```

Comment:
>--China--*/ [](start = 26, length = 11)

Can we also be more descriptive here with the comment? I would add a comment in the following line instead saying something like: "This cert should be used when connecting to Azure IoT on the Azure China Cloud (a.k.a. Mooncake)." And of course, something similar for the other certs including Baltimore. It would help customers to understand why 3 certs.

Reply:
are the code names public?

Reply:
Updated with comments below each cert name.